CHANGES/1639-misc

```diff
@@ -0,0 +1 @@
+Add exporter command for project wisdom
```

Comment:
@dmzoneill this PR will need a link to a Jira ticket in the AAH project. You can either move `AA-1624` to AAH or I can help create an AAH ticket linked to the AA one, just let me know.

Then this file can be renamed `####.misc` to match the AAH ticket number.

You'll also need to rebase this PR so it picks up our recent changes, which may include CI checks.